barman/retention_policies.py

```diff
@@ -126,6 +126,29 @@ def to_json(self):
         """
         return "%s %s %s" % (self.mode, self.value, self.unit)
 
+    def propagate_retention_status_to_children(self, backup_info, report, ret_status):
+        """
+        Propagate retention status to all backups in the chain.
```

Comment:
```suggestion
        Propagate retention status to all backups in the tree.
```

"Tree" sounds like a more appropriate word here. What do you think?
I would use "chain" when building a "single direction" path, with no options to go "left" or "right", if you get what I mean?

Reply:
Yeah, chain is linear indeed! Thanks